go.mod

```diff
@@ -1,4 +1,4 @@
-module github.com/bwmarrin/discordgo
+module github.com/EliasStar/discordgo
```

Comment:
```suggestion
module github.com/bwmarrin/discordgo
```

Reply:
whoops, didn't know pull requests reference  HEAD. thought they reference the latest commit at the time of creation. my bad🤦